bin/start_fe.sh

```diff
@@ -88,8 +88,11 @@ fi
 if [ ${ENABLE_DEBUGGER} -eq 1 ]; then
     # Allow attaching debuggers to the FE process:
     # https://www.jetbrains.com/help/idea/attaching-to-local-process.html
-    final_java_opt="${final_java_opt} -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5005"
-    echo $final_java_opt
+    final_java_opt="${final_java_opt} -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005"
+    if [[ "$JAVA_VERSION" -gt 8 ]]; then
+        final_java_opt="${final_java_opt} -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5005"
```

Comment:
if jave_version is greater than 8, we will append these debug options to final_java_opt twice?

Reply:
fixed